tests/test_lib_tokens_totp.py

```diff
@@ -499,11 +499,14 @@ def test_19_pin_otp_functions(self):
         #self.assertTrue(res == 47251647, res)
         self.assertTrue(res == -1, res)
 
-        # simple get_otp of current time
-        r = token.get_otp()
-        self.assertTrue(r > 47251648, r)
-        r = token.get_otp(current_time=datetime.datetime.now())
-        self.assertTrue(r > 47251648, r)
+        # simple OTPs of current time
+        # TODO: Not sure if that's what should be tested here
```

Comment:
This is a bit tricky here. The assertion ``r > 47251648`` apparently always succeeded in Python 2, because ``r`` is a tuple:
```python
>>> (1, "pin", "otp", "pinotp") > 47251648
True
```
But under Python 3, tuples and ints are incomparable:
```python
>>> (1, "pin", "otp", "pinotp") > 47251648
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: '>' not supported between instances of 'tuple' and 'int'
```
so the test above fails.

So I tried to find out what should be tested here: If I understand correctly, the test was intended to check that the TOTP counter value is bigger than 47251648. I tried to fix the test accordingly, but I'm not sure if it makes sense. @cornelinux you got any idea?

Reply:
Yes, it is probably the OTP counter. I also stumbled upon this at another python3 PR I issued some days ago.

Reply:
OK thanks, then I'll just keep the updated test and remove the TODO comment.